libexec/bats-core/bats-exec-test

```diff
@@ -40,24 +40,21 @@ fi
 BATS_TEST_DIRNAME="${BATS_TEST_FILENAME%/*}"
 BATS_TEST_NAMES=()
 
+# Shorthand for source-ing files relative to the BATS_TEST_DIRNAME,
+# optionally with a .bash suffix appended. If the argument doesn't
+# resolve relative to BATS_TEST_DIRNAME it is sourced as-is.
 load() {
-  local name="$1"
-  local filename
-
-  if [[ "${name:0:1}" == '/' ]]; then
-    filename="${name}"
-  else
-    filename="$BATS_TEST_DIRNAME/${name}.bash"
-  fi
+  local file="${1:?name}"
 
-  if [[ ! -f "$filename" ]]; then
-    printf 'bats: %s does not exist\n' "$filename" >&2
-    exit 1
```

Comment:
Could we re-add this error message please? So users don't notice the change : ]

Reply:
Do we actually expect any users to notice this message change? 😄 You'd typically only see it immediately before your test `exit`s due to a user error (such as assuming relative paths work...), I'm not sure anyone will miss this message. `source` prints an error when it fails to source a file.

That said, the motivation for removing it is to allow `source` to access files on the `PATH`, which `-f` precludes. I believe [something like](https://stackoverflow.com/a/6569837/113632):

    if [[ ! -e "$file" ]] && ! type -P "$file" >/dev/null; then

would suffice, but I was hoping to simplify the implementation by just deferring error-handling to `source`.

Reply:
It looks like the decision was made for us :) [Bash 3.2 seems unhappy with the `|| exit` syntax](https://travis-ci.org/github/bats-core/bats-core/jobs/678477480) (though I'm not exactly sure why), so I went with the approach mentioned above, which preserves the error message.